cremalab/sarah-todo
# Adasp 25 modal

## Description

I created a modal that is shown when the "Create New Task" button is clicked. When "cancel" is clicked, the modal is hidden.

### 🔗 Reference

> _Relevant links (e.g. issue, design, etc.)_

- [ADASP-25](https://crema.atlassian.net/jira/software/projects/ADASP/boards/108?selectedIssue=ADASP-25)

### 📋 Acceptance Criteria

> _Checked off by PR **Assignees**_

- [X] Clicking “Create New Task” button opens modal
- [X] Clicking “Save” button will add the new task to the Active Tasks section and close the modal
- [X] Clicking “Cancel” will close the modal with no changes to tasks
- [X] Placeholder text in input will say “Task description”
- [X] modal will have a title that says “New Task”






### 👩‍🔬 Test Instructions

> _Provided by PR **Assignees**_

1. Click "Create New Task"
2. Enter a task in the input with the placeholder test "Task description"
3. Click the "save" button
4. Task should appear under "Active Tasks"
5. Click "Create New Task"
6. Click "cancel" button
7. Modal should be hidden

## 🔎 Reviewer Checklist

> _Checked off by PR **Reviewers**_




---

> ### Roles & Responsibilities
>
> #### 👨‍💻 Assignee
>
> - Initiator of this PR (be sure to set in GitHub UI)
> - Addresses feedback and change requests
> - Merges PR once approved (usually deletes branch unless `develop` or `release`)
>
> #### 👩‍💻 Reviewer
>
> - Invited to review PR by **Assignee** (via GitHub UI)
> - Is expected to complete a review and address followup
